text/0017-remove-adoptions.md

```diff
@@ -0,0 +1,93 @@
+- Feature Name: remove\_adoptions
+- Start Date: 2024-11-22
+- RFC PR: (leave this empty)
+- Bundler Issue: (leave this empty)
+
+# Summary
+
+Remove the Adoptions feature from RubyGems.org due to its high maintenance burden and limited benefit.
+
+# Motivation
+
+Adoptions aimed to address the lack of maintenance in older Ruby gems but has proven unsuccessful.
+
+Implementing Pundit authorization and Organizations features for RubyGems.org highlighted this issue. About 50% of my authorization efforts were spent handling the complexity of adoptions.
+
+However, the feature's low effectiveness does not justify this effort.
+
+- 45 "Ownership Calls" (requests for new maintainers) created.
+- 140 "Ownership Requests" submitted:
+  - 40 tied to Ownership Calls, with only 6 approved.
+  - 100 unsolicited, with 63 ignored, 8 approved, and 29 denied.
+
+Only 14 successful ownership transfers occurred, reflecting a mismatch between the low-trust adoption process and the high-trust requirements of gem ownership.
```

Comment:
:bar_chart: 14 successful ownerships in ~2.5 years, half of related gems are still abandoned and one gem was just fully yanked and never released again